nix/keys.nix

```diff
@@ -133,6 +155,8 @@ in
               let
                 # FIXME: The key file should be marked as private once
                 # https://github.com/NixOS/nix/issues/8 is fixed.
+                #
+                # FIXME: implement value.filename for this as well
```

Comment:
This will fail (by the virtue of the assertions now moved to Nix) if _neither_ `text` nor `filename` is set is that good enough?